python/taichi/lang/impl.py

```diff
@@ -300,7 +300,13 @@ def layout(func):
 
 
 def ti_print(*vars):
-    taichi_lang_core.create_print([Expr(var).ptr for var in vars])
+    def load_if_expr(var):
+        if isinstance(var, str):
+            return var
+        else:
+            return Expr(var).ptr
+
+    taichi_lang_core.create_print([load_if_expr(var) for var in vars])
```

Comment:
```suggestion
    taichi_lang_core.create_print([Expr(var).ptr for var in vars])
```

Reply:
Then how do I take care if `isinstance(var, str)`?
Also note that the reason why I want a `load_if_expr` is for matrix print (iapr), which requires `load_if_expr` to return a list of it's elements `['[', m[0], ', ', m[1], ']']`

Reply:
I see. I was confused by the function name `load_if_expr` - how about renaming it to `expand`? (People may wonder what we are **loading** here...)